apps/mymedicare_cb/authorization.py

```diff
@@ -280,16 +280,25 @@ def service_health_check(self, request):
         the BB2 /health/external check.
         """
         headers = self.slsx_common_headers(request)
-
-        response = requests.get(
-            self.healthcheck_endpoint,
-            headers=headers,
-            allow_redirects=False,
-            verify=self.verify_ssl_internal,
-            timeout=5,
-        )
-        response.raise_for_status()
-        return True
+        max_retries = 3
+        retries = 0
+        while retries < max_retries:
+            try:
+                response = requests.get(
+                    self.healthcheck_endpoint,
+                    headers=headers,
+                    allow_redirects=False,
+                    verify=self.verify_ssl_internal,
+                    timeout=10,
```

Comment:
Oh actually was wondering about this, was there a reason to increase the timeout to 10 instead of 5? I'm cool with it either way, just wanted to make sure this wasn't by mistake.

Reply:
Still didn't see a response here, any thoughts?